### ISSUE TITLE
Reading settings crashes Aedict

### ISSUE THREAD

**Issue by mvysny** [mvysny/aedict#711]
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{sk.baka.aedict3/sk.baka.aedict3.ConfigActivity}: java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.String
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3253)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3349)
	at android.app.ActivityThread.access$1100(ActivityThread.java:221)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1794)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:158)
	at android.app.ActivityThread.main(ActivityThread.java:7225)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
Caused by: java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.String
	at android.app.SharedPreferencesImpl.getString(SharedPreferencesImpl.java:225)
	at android.preference.Preference.getPersistedString(Preference.java:1706)
	at android.preference.ListPreference.onSetInitialValue(ListPreference.java:295)
	at android.preference.Preference.dispatchSetInitialValue(Preference.java:1622)
	at android.preference.Preference.onAttachedToHierarchy(Preference.java:1416)
	at android.preference.PreferenceGroup.addPreference(PreferenceGroup.java:175)
	at android.preference.PreferenceGroup.addItemFromInflater(PreferenceGroup.java:116)
	at android.preference.PreferenceGroup.addItemFromInflater(PreferenceGroup.java:42)
	at android.preference.GenericInflater.rInflate(GenericInflater.java:490)
	at android.preference.GenericInflater.rInflate(GenericInflater.java:495)
	at android.preference.GenericInflater.rInflate(GenericInflater.java:495)
	at android.preference.GenericInflater.inflate(GenericInflater.java:327)
	at android.preference.GenericInflater.inflate(GenericInflater.java:264)
	at android.preference.PreferenceManager.inflateFromResource(PreferenceManager.java:274)
	at android.preference.PreferenceActivity.addPreferencesFromResource(PreferenceActivity.java:2069)
	at sk.baka.aedict3.ConfigActivity.onCreate(ConfigActivity.java:115)
	at android.app.Activity.performCreate(Activity.java:6876)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1135)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3206)
	... 9 more
```

**Comment by mvysny** [comment]
https://code.google.com/p/android/issues/detail?id=37387
Apparently Google thinks this bug is Obsolete. Even though on my console it clearly shows that a lot of Android 6.0 phones suffer from this issue. OMG. Android is such a crap. The only thing that keeps me in this pile of shit is that it is the only widespread mobile phone vendor which has language with a GC.

**Comment by mvysny** [comment]
Oh, I tried to store Integers as Integers in Android Preferences. I can't do that on Android, oh silly me!
Workaround implemented in Aedict 3.39.27.
